src/client/actions/GuildMemberUpdate.js

```diff
@@ -0,0 +1,44 @@
+'use strict';
+
+const Action = require('./Action');
+const { Status, Events } = require('../../util/Constants');
+
+class GuildMemberUpdate extends Action {
+  handle(data, shard) {
+    const { client } = this;
+    const user = client.users.cache.get(data.user.id);
+    if (data.user.username) {
```

Comment:
`user` is used only inside the condition:
```suggestion
    if (data.user.username) {
      const user = client.users.cache.get(data.user.id);
```